config/config.go

```diff
@@ -143,7 +144,7 @@ func (c *Config) UpdateTempStoragePath() {
 
 func encodeDefTempStorageDir(port, statusPort uint) string {
 	dirName := base64.URLEncoding.EncodeToString([]byte(fmt.Sprintf("%v/%v", port, statusPort)))
-	return filepath.Join(os.TempDir(), "tidb", dirName, "tmp-storage")
+	return filepath.Join(os.TempDir(), strconv.Itoa(os.Getuid())+"_tidb", dirName, "tmp-storage")
```

Comment:
`os.Getuid()` returns -1 on Windows, see https://golang.org/pkg/os/#Getuid. An alternative try is `os/user.Current()`, and it's better to add a test case for `encodeDefTempStorageDir` and cover it in both platform.